packages/bull/lib/interfaces/bull-module-options.interface.ts

```diff
@@ -25,6 +21,11 @@ export interface BullModuleOptions extends Bull.QueueOptions {
    * Additional queue processors
    */
   processors?: BullQueueProcessor[];
+
+  /**
+   * Redis client connection string
+   */
+  url?: string;
```

Comment:
Note: adding this isn't sufficient. `forRoot` still won't let you register a configuration object with an `url` property.

Reply:
I see, I can add a `BullRootModuleOptions` that extends `Bull.QueueOptions` and contains the url.

Reply:
@kamilmysliwiec Is the fix ok? I would really like to have this feature.